open-company/open-company-web
# Dark mode

Card: https://trello.com/c/wRV1c4fd

To test:
- check all views in light mode
- check all views in dark mode
- check Carrot inherit the dark mode from system when in auto
- check that you can override the system mode from the setting panel
- check that you can go back to auto mode from the setting panel
